src/routers/filing.py

```diff
@@ -78,3 +78,17 @@ async def patch_filing(request: Request, lei: str, period_name: str, field_name:
 @requires("authenticated")
 async def update_task_state(request: Request, lei: str, period_name: str, task_name: str, state: StateUpdateDTO):
     await repo.update_task_state(request.state.db_session, lei, period_name, task_name, state.state, request.user)
+
+
+@router.get("/institutions/{lei}/filings/{period_name}/contact_info", response_model=List[ContactInfoDTO])
+@requires("authenticated")
+async def get_contact_info(request: Request, lei: str, period_name: str):
+    return await repo.get_contact_info(request.state.db_session, lei, period_name)
+
+
+@router.post("/institutions/{lei}/filings/{period_name}/contact_info", response_model=ContactInfoDTO)
+async def post_contact_info(request: Request, lei: str, period_name: str, contact_info_obj: ContactInfoDTO = None):
+    if contact_info_obj:
+        return await repo.update_contact_info(request.state.db_session, contact_info_obj)
```

Comment:
we'll need to have the method aware of the `lei`, and `period_name` parameters as well; otherwise, we won't be able to associate the contact_info to the correct filing.

Reply:
Thanks, working on it.